src/contracts/core/AllocationManagerStorage.sol

```diff
@@ -83,6 +83,8 @@ abstract contract AllocationManagerStorage is IAllocationManager {
         _maxMagnitudeHistory;
 
     /// @dev For a strategy, contains the amount of magnitude an operator has allocated to operator sets
+    /// @dev This value should be read with caution, as deallocations that are completable but not
+    ///      popped off the queue are still included in the encumbered magnitude
     mapping(address operator => mapping(IStrategy strategy => uint64)) public encumberedMagnitude;
```

Comment:
Good clarification, the "true" encumberedMagnitude can be read from calling 
`getMaxMagnitude(operator, strategy) - getAllocatableMagnitude(operator, strategy)` which could potentially give you a smaller value than what is in storage.